python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/traffic_policies.schema.yml

```
@@ -192,8 +192,9 @@ keys:
                         convert_types:
                           - int
                         description: |-
-                          Counter name. This should also be added to the `policies[].counters` list.
-                          It will no longer be added automatically to the counters in AVD 6.0.
+                          Named counter.
+                          Must also be defined under `policies[].counters` to be active.
+                          If not defined, the counter will be inactive on EOS and a warning is printed via the 'show traffic-policy interface' command.
```

lets not talk about the warning. Same below
```suggestion
                          If not defined, the counter is inactive on EOS.
```